apps/cwl_firehose_subscriber/main.tf

```diff
@@ -91,28 +91,22 @@ EOF
 depends_on = ["aws_iam_role.cwl_firehose_subscriber"]
 }
 
-data "archive_file" "lambda_zip" {
-  type = "zip"
-  source_dir = "./target"
-  output_path = "./lambda.zip"
-}
-
 resource "aws_lambda_function" "cwl_firehose_subscriber" {
-  filename = "${data.archive_file.lambda_zip.output_path}"
   function_name = "cwl_firehose_subscriber"
+  s3_bucket = "org-hca-logs-lambda-deployment"
+  s3_key = "cwl-firehose-subscriber/CWLFirehoseSubscriber_App.zip"
```

Comment:
nit: it would be nice if there was one source of truth for the s3_bucket name and s3_key
* for the bucket this might be a little tricky; perhaps the config is the source of truth and generates the bucket?
* for the key, this is easy; you could just pull this in from the `Makefile`

Reply:
👍

Reply:
👍